plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -0,0 +1,241 @@
+          </FormGroup>
+          <FormGroup
+            className="column-3"
+            required={false}
+            showError={Boolean(errors.healthChecks[key])}>
+            <FieldLabel>Max Failures</FieldLabel>
+            <FieldInput
+              name={`healthChecks.${key}.maxConsecutiveFailures`}
+              type="number"
+              min="0"
+              value={healthCheck.maxConsecutiveFailures}/>
+            <FieldError>{errors.healthChecks[key]}</FieldError>
+          </FormGroup>
+        </div>
+      </CollapsibleContainer>);
+  }
+
+  getCommandFields(healthCheck, key) {
+    const {errors} = this.props;
+
+    if (healthCheck.protocol !== 'COMMAND') {
+      return null;
+    }
+
+    return (
+      <div className="flex row">
+        <FormGroup
+          className="column-12"
+          required={false}
+          showError={Boolean(errors.healthChecks[key])}>
+          <FieldLabel>Command</FieldLabel>
+          <FieldTextarea
+            name={`healthChecks.${key}.command`}
+            type="text"
+            value={healthCheck.command}/>
+          <FieldError>{errors.healthChecks[key]}</FieldError>
+        </FormGroup>
+      </div>
+    );
+  }
+
+  getHTTPFields(healthCheck, key) {
+    const {errors} = this.props;
+
+    if (healthCheck.protocol !== 'HTTP' && healthCheck.protocol !== 'HTTPS') {
+      return null;
+    }
+
+    return [
+      (<div className="flex row" key="path">
+        <FormGroup
+          className="column-6"
+          required={false}
+          showError={false}>
+          <FieldLabel>Service Endpoint</FieldLabel>
+          <FieldSelect name={`healthChecks.${key}.portIndex`}>
+            <option value="">Select Endpoint</option>
+          </FieldSelect>
+        </FormGroup>
+        <FormGroup
+          className="column-6"
+          required={false}
+          showError={Boolean(errors.healthChecks[key])}>
+          <FieldLabel>Path</FieldLabel>
+          <FieldInput
+            name={`healthChecks.${key}.path`}
+            type="text"
+            value={healthCheck.path}/>
+          <FieldError>{errors.healthChecks[key]}</FieldError>
+        </FormGroup>
+      </div>),
+      (<div className="row flex" key="HTTPS">
+        <FormGroup showError={false} className="column-12">
+          <FieldLabel>
+            <FieldInput
+              checked={healthCheck.protocol === 'HTTPS'}
+              name={`healthChecks.${key}.https`}
+              type="checkbox"
+              value="HTTPS"/>
+            Make HTTPS
+            <FieldHelp>Morbi leo risus</FieldHelp>
+          </FieldLabel>
+          <FieldError>{errors.healthChecks[key]}</FieldError>
+        </FormGroup>
+      </div>)
+    ];
+  }
+
+  getHealthChecksLines(data) {
+    const {errors} = this.props;
+
+    return data.map((healthCheck, key) => {
+      return (
+        <div key={key} className="panel pod-short">
+          <div className="pod-narrow pod-short">
```

Comment:
I think we should create a simple component for this. Something like `FormGroupContainer` or something to indicate that it is actually a group of form fields. Then maybe it can have a prop like `removable`, and when `removable={true}`, the component displays the `form-remove` element (of course it would have to accept an `onRemove` prop too). I think this would help us be more consistent. What do you think?